File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/detailspackage/ProfileModel/ProfilePojo.java
package com.example.esraarashad.httpurlconnectionexample.detailspackage.ProfileModel;

import java.util.ArrayList;

public class ProfilePojo {
    private ArrayList<Profiles> profiles;

    private int id;

    public ArrayList<Profiles> getProfiles ()
    {
        return profiles;
    }

    public void setProfiles (ArrayList<Profiles>  profiles)
    {
        this.profiles = profiles;
    }

    public int getId ()
    {
        return id;
    }

    public void setId (int id)
    {
        this.id = id;
    }

    @Override
    public String toString()
    {
        return "ClassPojo [profiles = "+profiles+", id = "+id+"]";
    }
}

File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/detailspackage/model/ProfileModel/ProfilePojo.kt
package com.example.esraarashad.httpurlconnectionexample.detailspackage.model.ProfileModel

import java.util.ArrayList

class ProfilePojo {
    var profiles: ArrayList<Profiles>? = null

    var id: Int = 0

    override fun toString(): String {
        return "ClassPojo [profiles = $profiles, id = $id]"
    }
}
